rest_framework_json_api/utils.py

```diff
@@ -40,7 +40,7 @@ def get_resource_name(context, expand_polymorphic_types=False):
 
     # Sanity check to make sure we have a view.
     if not view:
-        raise APIException(_('Could not find view.'))
```

Comment:
Why was this switched to returning None instead of raising an exception?

Reply:
DRF uses the test renderer to parse data passed on to `APIClient`. In such a case it won't have a renderer context and therefore there is no view provided.
The json api renderer should still not fail in such a case and render data as is. Hence I had to remove this exception.
Interesting enough is though that the json api renderer was already aware of `resource_name` with value `None`.

Reply:
ok, if I understand this correctly based on what you wrote, missing a view should not be possible in a non-test setting? Does that sound true?

Reply:
yep this is true...